A11yUITests/Classes/A11yTests.swift

```diff
@@ -0,0 +1,159 @@
+//
+//  A11yTests.swift
+//  A11yUITests
+//
+//  Created by Rob Whitaker on 26/03/2021.
+//
+
+import XCTest
+
+internal extension XCTestCase {
+
+    func a11yCheckValidSizeFor(element: A11yElement,
+                               file: StaticString,
+                               line: UInt) {
+
+        guard !element.shouldIgnore else { return }
+
+        XCTAssert(element.frame.size.height >= 18,
```

Comment:
There is `XCTAssertGreaterThanOrEqual()` and friends that may give better error messages than the plain `XCTAssert` here.

Reply:
Nice tip, thanks Bas